certification/internal/policy/operator/deployable_by_olm.go

```diff
@@ -41,21 +44,57 @@ type OperatorData struct {
 }
 
 type DeployableByOlmCheck struct {
-	OpenshiftEngine   cli.OpenshiftEngine
 	OperatorSdkEngine cli.OperatorSdkEngine
+	openshiftEngine   cli.OpenshiftEngine
```

Comment:
We've got a symbol overloading problem here. It's not directly to your change so consider this non-blocking. Just noting for awareness. `cli.openshiftEngine` (interface) vs. engine.openshiftEngine (impl). My gut instinct was to ask why we have a client key in the next line if openshiftEngine has a client, but then I saw the distinction.

Reply:
Actually, I think that question is still valid, specifically because NewDepoyableByOLM(...) doesn't seem to accept an openshiftEngine?

Reply:
It does not. In fact, OpenshiftEngine is another "it should have never been called that". It's really just a wrapper around k8s client calls. I want to move it (along with openshift_version.go) into its own package. Open to suggestions.